pkg/util/testingjobs/rayjob/wrappers.go

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rayjob
+
+import (
+	rayjobapi "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/kueue/pkg/controller/jobframework"
+	"sigs.k8s.io/kueue/pkg/util/pointer"
+)
+
+// JobWrapper wraps a RayJob.
+type JobWrapper struct{ rayjobapi.RayJob }
```

Comment:
Uhm.. I think `RayJobWrapper` would be good.
@trasc wdyt?

Reply:
Makes sense.
Maybe, we need to change `MakeMPIJob` to `MakeJob` as well.

https://github.com/kubernetes-sigs/kueue/blob/74f45f69612a2d36ace91ae455f1e9fdbcd53037/pkg/util/testingjobs/mpijob/wrappers_mpijob.go#L34

We can work on this at follow-up PRs.

Reply:
sure